docs/central-entities.rst

```diff
@@ -137,6 +156,44 @@ To try the end-to-end workflow across multiple Forms, we recommend creating a te
+
+To get started, navigate to the Entity List you'd like to load your data into, and select the :guilabel:`Data` tab. At the top of that screen, click on the :guilabel:`Upload` button.
+
+.. image:: /img/central-entities/entity-upload.png
+
+You should see a screen that looks similar to the above. You can see data tables with column headings matching the properties in your Entity List. If you have any Entities in this Entity List already, you will see a preview of them in the table above.
+
+To add your new data to this Entity List, you will first need to convert it into a ``.csv`` file if it is not already. You can use common spreadsheet applications like Excel, Google Sheets, or OpenOffice Calc to do this.
+
+There are some additional requirements on this data file:
+
+1. The first row (and *only* the first row) is a header row labeling each of your columns.
+2. Your header row must exactly match your Entity List properties.
+3. In addition to your Entity List properties, you must also include a ``label`` column, where you can provide the name for each Entity that is used when displaying the Entity in Central and Collect.
+
+If you're not sure what this should look like, you can download a sample template ``.csv`` file by clicking the :guilabel:`Download a Data Template` button.
+
+.. note::
+  *What if my Entity List doesn't have any properties yet? Can't you just read them from my file?*
+
+  Not yet. Please `let us know <https://forum.getodk.org/c/ideas/9>`_ if this is something you would like to see! For now, you will need to add the properties manually. The quickest way to do this is :ref:`in the Entity Properties section <central-entities-creating-direct>` in Central.
+
+Once you have a ``.csv`` file that meets these requirements, you can select it for upload either by dragging it over the box, or by clicking on the ``choose one`` button. If your file doesn't meet the above requirements, or if Central can't understand how to read the file, you will see an error message appear. Otherwise, you will see a preview of the data that Central found in the file and how it will load into the Entity List.
+
+.. image:: /img/central-entities/entity-upload-preview.png
+
+Looking at this view, you can match up the columns and ensure your data is loading into the correct places. You can also verify that the number of records in your file matches the number of Entities Central found to import. It is also often a good idea to check at least the last few rows of your import data in the preview table in addition to the beginning. If there is anything Central is worried about, you will see warnings above your import data. Warnings at this point mean that Central is able to import your file, but it isn't completely sure it's interpreted your file correctly. If you don't see any problems looking over the preview, you can feel safe to ignore the warnings.
```

Comment:
Nit: I stumbled a bit on "it isn't completely sure it's interpreted your file correctly" -- I think "it has" instead of "it's" here would make for smoother reading (I understood "it is" first).